lib/teams/emrContainersRole.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: emr-containers
+  namespace: <REPLACE-NAMESPACE>
```

Comment:
let's rename these to .ytpl (yaml template) otherwise the quickstart code scanner will flag the yaml for the replaceable parameter. We do it elsewhere, e.g in amp add-on.